public-api/src/main/java/org/pmiops/workbench/service/CdrVersionService.java

```diff
@@ -0,0 +1,31 @@
+package org.pmiops.workbench.service;
+
+import org.pmiops.workbench.cdr.CdrVersionContext;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import javax.inject.Provider;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.pmiops.workbench.db.model.CdrVersion;
+import org.pmiops.workbench.db.dao.CdrVersionDao;
+
+@Service
+public class CdrVersionService {
+    private CdrVersionDao cdrVersionDao;
+
+    @Autowired
+    public CdrVersionService(CdrVersionDao cdrVersionDao) {
+        this.cdrVersionDao = cdrVersionDao;
+    }
+
+    @Autowired
+    @Qualifier("defaultCdr")
+    private Provider<CdrVersion> defaultCdrVersionProvider;
+
+    public void setDefaultCdrVersion() {
+        CdrVersionContext.setCdrVersionNoCheckAuthDomain(defaultCdrVersionProvider.get());
+    }
+
+    public CdrVersion findByIsDefault(boolean isDefault) {
```

Comment:
So it might be a good idea to follow the pattern on workbench... where any database models are prefixed with Db and client models stay the same. For example we call the database model object DbCdrVersion and the client model CdrVersion. This way it is clear which is which

Reply:
Changed Cdrversion model for now. Will change rest of them in coming prs.